target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -262,6 +262,14 @@ m2m)
 	set_wifi_led "$boardname:blue:wifi"
 	ucidef_set_led_netdev "eth" "Ethernet" "$boardname:green:wan" "eth0"
 	;;
+mikrotik,rbm11g)
+	ucidef_set_rssimon "wlan0" "200000" "1"
+	ucidef_set_led_rssi "rssilow" "RSSILOW" "rbm11g:green:rssi0" "wlan0" "26" "100" "-25" "13"
```

Comment:
Please use "$boardname:green:..." for the LEDs.

Reply:
Do you have pwm leds (LEDs which can have different brightness level)? If not the last two values can be omitted.

I'm somehow in doubt that you checked the rssi led syntax. It is:

`... <led> <wireless interface> <signal quality min> <signal quality max> ...`

With five rssi LEDs I rather would expect a configuration like:

```
ucidef_set_led_rssi "rssilow" "RSSILOW" "$boardname:green:rssi0" "wlan0" "1" "100"
ucidef_set_led_rssi "rssimediumlow" "RSSIMEDIUMLOW" "$boardname:green:rssi1" "wlan0" "20" "100"
ucidef_set_led_rssi "rssimedium" "RSSIMEDIUM" "$boardname:green:rssi2" "wlan0" "40" "100"
ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "$boardname:green:rssi3" "wlan0" "60" "100"
ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssi4" "wlan0" "80" "100"
```

rssi0 is on if the quality is between 1% and 100%
rssi1 is on if the quality is between 20% and 100%

and so on.

Reply:
Your doubts are well founded :smile: . I'll definetly fix that